Base.rte/Activities.ini

```diff
@@ -11,6 +11,20 @@ AddActivity = GATutorial
 	Team1Funds = 1500
 	CPUTeam = 1
 	MaxPlayerSupport = 1
+	
+
+AddActivity = GAScripted
+	PresetName = Test Activity
```

Comment:
I know this is convenient but move it back to the end.
Eventually:tm: it won't even be part of vanilla.

Reply:
Booooooooo

Fine

Reply:
Fine shmine it's still not back where it was. Now it's stuck in the middle of other activities. 😠